packages/components/src/inline-edit/README.MD

```diff
@@ -0,0 +1,6 @@
+Hook providing with data to bend two elements.
+
+One of which sets 'isEdit' state to true upon clicking, second sets 'isEdit' state to false on 'Enter'/onBlur event and triggering onCommit callback with its value.
+
+
+The hook was designed to be a component in the first place, but hook gives more flexibility. See more: https://github.com/WordPress/gutenberg/pull/28924
```

Comment:
"The hook was designed to be a component in the first place, but **a** hook gives more flexibility."